applications/minotari_app_grpc/proto/base_node.proto

```diff
@@ -591,20 +594,24 @@ message SearchPaymentReferencesRequest {
 message PaymentReferenceResponse {
     // The payment reference that was found
     string payment_reference_hex = 1;
-    // Block height where the output was mined
+    // Block height where the output was mined - will be 0 for spent outputs
     uint64 block_height = 2;
-    // Block hash where the output was mined
+    // Block hash where the output was mined - will be empty for spent outputs
     bytes block_hash = 3;
-    // Timestamp when the output was mined
+    // Timestamp when the output was mined - will be 0 for spent outputs
     uint64 mined_timestamp = 4;
-    // Output commitment (32 bytes)
+    // Output commitment (32 bytes) - will be empty if output was spent
     bytes commitment = 5;
     // Whether this output has been spent
     bool is_spent = 6;
-    // Height where output was spent (if spent)
+    // Height where output was spent - will be 0 for unspent outputs
     uint64 spent_height = 7;
-    // Block hash where output was spent (if spent)
+    // Block hash where output was spent - will be empty for unspent outputs
     bytes spent_block_hash = 8;
-    // Transaction output amount will be 0 for non set a
+    // Minimum value promise - will be 0 for spent outputs
```

Comment:
I dont think this comment is correct?

Reply:
We populate it with 0 for spent outputs as we do not know what it is